ocrd_monitor/init.sh

```diff
@@ -18,4 +18,5 @@ if [ -n "$CONTROLLER" ]; then
 fi
 
 broadwayd :5 &
-python -u serve.py -p 8080 -P 8085 -d "${1:-/data}"
+
+flask --app /usr/local/ocrd-monitor/app --debug run --host='0.0.0.0'
```

Comment:
We still need to pass the actual data path here (because it is more flexible than a fixed /data). Should become `BASEDIR` somehow, I guess

Reply:
BTW, I'm solving this in the next PR by simply chdiring to `$1` first, so in the app all paths can be relative.

Reply:
Ok in this PR I added an environment var to configure the dir. Relative path without using this basedir in parameters sound good.